### DISCUSSION TITLE
make it easier to connect with DIY adapters

### DISCUSSION THREAD

**Discussion by kleffa** [darthcloud/BlueRetro#639] [Ideas]
Blueretro can be set up to 4 controllers like NGC, but PSX/2 which has more buttons only can configured 2 of them.
There is anyway to set serial communication or get the PSX/2 players 3 and 4 data via pin like the NGC controllers... including home button if posibble?

I noted most adapters used with blueretro are PSX ones because amount of buttons, but a serial conection with the 4 controllers data should make it easy to make Arduino adapters to PS3/Xbox/pc/switch

**Comment by darthcloud** [comment]
Set BlueRetro in multitap mode.

PSX multitap expose 4 players on each port.

**Reply by kleffa** [reply]
Was not supposed multitap doesn't work since esp32 is not enough fast? Sorry, I get lost in that

**Reply by darthcloud** [reply]
PS1 multitap is supported (as stated in the wiki) what is not supported is PS2 multitap.
